core/domain/subtopic_validators_test.py

```diff
@@ -0,0 +1,1003 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core.domain.subtopic_validators."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+
+from constants import constants
+from core.domain import prod_validation_jobs_one_off
+from core.domain import skill_domain
+from core.domain import skill_services
+from core.domain import story_domain
+from core.domain import story_services
+from core.domain import subtopic_page_domain
+from core.domain import topic_domain
+from core.domain import topic_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+import python_utils
+
+datastore_services = models.Registry.import_datastore_services()
+
+USER_EMAIL = 'useremail@example.com'
+USER_NAME = 'username'
+
+(
+    story_models, subtopic_models,
```

Comment:
Do we need `story_models` here?

Reply:
Yes, without this can lead to a failure in backend tests.

Reply:
I don't see any place where `story_models` is used in this file. can you please share the error log?

Reply:
<img width="1180" alt="Screenshot 2021-02-05 at 8 49 27 AM" src="https://user-images.githubusercontent.com/73252546/106986766-10d1c080-6792-11eb-990d-683634069b65.png">
And the same applies for removal of topic_model in subtopic_validators.py which was mentioned above @DubeySandeep

Reply:
The errors are related to story_domain class not defined (I believe you have removed the story_domain import), but I'm asking to remove story_models. Can you please check after only removing the story_models?

Reply:
Yes, you are right the change was due to removal of story_domain and also story_models was also removed 
@DubeySandeep PTAL